stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1172,6 +1172,65 @@ public func _TFCGetCTensorHandleFromSwift(
   return handle.cTensorHandle
 }
 
+/// Adds `handle` as an input to `op`.
+@usableFromInline
+@_silgen_name("_swift_tfc_OpAddInputFromTensorHandle")
+func _TFCOpAddInputFromTensorHandle(_ op: CTFEOp,
+                                    _ handle: _AnyTensorHandle,
+                                    _ status: CTFStatus) {
+  TFE_OpAddInput(op, handle.cTensorHandle, status)
+}
+
+/// Adds `t` as an input or inputs to `op`.
+@usableFromInline
+@_silgen_name("_swift_tfc_OpAddInputFromTensorGroup")
+func _TFCOpAddInputFromTensorGroup<T: TensorGroup>(
+    _ op: CTFEOp, _ t: T, _ status: CTFStatus
+) -> Int32 {
+  let count = T._tensorHandleCount
```

Comment:
Since the `Int32` version of `count` is never used in this function (as far as i can tell), you might as well convert it to `Int` here and drop all other conversions below.